resources/sql/operations/upload-file.sql

```diff
@@ -1,11 +1,11 @@
-INSERT INTO files (filename, filepath, app_id, owner_id, contents)
+INSERT INTO files (filename, filepath, app_id, owner_id, file_size)
 VALUES (
     '%{filename}',
     '%{filepath}',
     %{app_id},
     %{user_id},
-    E'%{contents}'
+    %{file_size}
```

Comment:
SQL code should probably be parameterized or at least be properly sanitized for sqlite

Reply:
This will be handled on #156